src/main/java/spireMapOverhaul/util/ActUtil.java

```diff
@@ -1,11 +1,40 @@
 package spireMapOverhaul.util;
 
+import com.evacipated.cardcrawl.modthespire.Loader;
 import com.megacrit.cardcrawl.core.Settings;
-import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.dungeons.*;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
 
 public class ActUtil {
     public static int getRealActNum() {
```

Comment:
This implementation is adapted from what I did for Corrupt the Spire. I found it to be a reliable way of getting the real act number -- haven't had any errors related to it since I implemented it.